inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_convolution_test.cpp

```diff
@@ -424,7 +424,7 @@ INSTANTIATE_TEST_CASE_P(accuracy_group, myriadLayerConvolution_smoke,
           )
 );
 
-INSTANTIATE_TEST_CASE_P(accuracy_group_large_input, myriadLayerConvolution_smoke,
+INSTANTIATE_TEST_CASE_P(accuracy_group_large_input, myriadLayerConvolution,
```

Comment:
Test group below this defined at line 442 still in pre-commit, but tests ChannelMinor as well. I guess this tests pack needs to be separated into 2 where only one is in pre-commit